gpii/node_modules/processReporter/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "processReporter",
+  "description": "The Process Reporter module provides information about the platform and its running software",
+  "version": "0.1.0",
+  "author": "GPII",
+  "bugs": "http://issues.gpii.net/browse/GPII",
+  "homepage": "http://gpii.net/",
+  "dependencies": {},
+  "licenses": [
+     {
+       "type": "BSD-3-Clause",
+       "url": "http://www.opensource.org/licenses/BSD-3-Clause"
+     }
+  ],
+  "keywords": ["gpii", "accessibility", "settings", "fluid", "IoC", "Inversion of Control", "configuration", "evented"],
+  "repository": "git://github.com/GPII/universal.git",
+  "main": "./index.js",
+  "engines": { "node" : ">=0.8" }
```

Comment:
Done.  Also, I updated similarly the package.json in the linux and windows pulls.